apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/BulkApiLiveTest.java

```diff
@@ -69,7 +69,7 @@ public void testExtractArchive() throws Exception {
                                                      .extractArchive(containerName, payload, "tar.gz");
          assertEquals(extractResponse.getCreated(), OBJECT_COUNT);
          assertTrue(extractResponse.getErrors().isEmpty());
-         assertEquals(api.getContainerApi(regionId).get(containerName).getObjectCount(), OBJECT_COUNT);
+         assertEquals(api.getContainerApi(regionId).get(containerName).getObjectCount(), Long.valueOf(OBJECT_COUNT));
```

Comment:
I left my comment at the wrong line. Sorry! There were a few Long.valueOf() such as 

> .objectCount(Long.valueOf(2))

that I think are not necessary, as long would be converted to Long automatically in that case.